README.rst

```diff
@@ -192,6 +192,30 @@ instance type.
     mxnet_estimator.delete_endpoint()
 
 
+If you have an existing model and would like to deploy it locally you can do that as well. If you don't
+specify a sagemaker_session argument to the MXNetModel constructor, the right session will be generated
+when calling model.deploy()
```

Comment:
This still sounds like a choice to me - shouldn't we just instruct the user to not supply a session object? (or would there be a reason for a user to want to construct a local session separately?)

Reply:
this is similar to how the rest of the SDK works. they can provide their own Local Session if they want, with custom config settings.

Reply:
what is a better wording ?

Reply:
maybe something like "This requires a Local Session (maybe include a link to documentation?) to run. If you don't specify..." or "If you want to specify a sagemaker session, make sure it's a local session"